spec/spec_helper.rb

```diff
@@ -1,26 +1,38 @@
-require 'rubygems'
```

Comment:
@Dan33l For the initial modulesync, perhaps mark this file as unmanaged?  A future PR can update it to the one from modulesync_config.